lib/utils/idleState.ts

```diff
@@ -1,93 +1,18 @@
 import Observers, { ListenerFunction } from './listeners';
-
-/*
-  IdleState let you know if the application is active or idle. The application is considered idle if not user
-  interactions have been detected for a specific amount of time (timeout). To use IdleState, you first create an
-  IdleState instance, then you can either get the current state from the instance or add an observer to be called each
-  time the state changes.
-
-  When creating an instance, you can provide a timeout, by default timeout of 3 minutes is used. When done with the
-  IdleState instance, you must call the delete method to release it.
-
-  If several consumers create an idleState with the same timeout, only one instance is created. The instance is shared
-  between the consumers.
-
-  Syntax:
-    new IdleState();
-    new IdleState(timeout);
-
-    timeout: timeout in milliseconds, default is 3 minutes.
-
-  Properties:
-    IdleState.state:
-      current state as string, either 'active' or 'idle', read only.
-
-  Methods:
-    IdleState.addObserver(observer):
-      Add an observer to be called when the idle state changes. The observer receives the new state as parameter. You
-      can several observers.
-      Returns a function to be called to remove the observer.
-
-    IdleState.delete():
-      Release the IdleState instance. As the same instance can be shared with multiple consumers, it's important to call
-      delete() when done with it. But first, you should remove any observer you have added, else they will still be
-      called until all consumers have released the instance.
-
-  Usage:
-    import IdleState from 'utils/IdleState';
-
-    // Create an instance with a timeout of 2 minutes
-    const idleState = new IdleState(2 * 60 * 1000);
-
-    // Add an observer
-    const removeObserver = idleState.addObserver(state => {
-      if (state === 'active') {
-        // ... application is now active
-      } else if (state === 'idle') {
-        // ... application is now idle
-      }
-    });
-
-    // Read the current state
-    const currentState = idleState.state;
-
-    // When done, release the instance, but first remove the observer
-    removeObserver();
-    idleState.delete();
-*/
-
-import { isTest } from './environment';
 export const STATE_ACTIVE = 'active';
 export const STATE_IDLE = 'idle';
```

Comment:
You can probably delete these constants as I suspect they are no longer being used.

Reply:
✅   done, thank you 🙇‍♀️